OrbitGl/TimerTrack.h

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef TIMER_TRACK_H_
+#define TIMER_TRACK_H_
+
+#include <map>
+#include <memory>
+
+#include "BlockChain.h"
+#include "CallstackTypes.h"
+#include "EventTrack.h"
+#include "TextBox.h"
+#include "Threading.h"
+#include "TimerChain.h"
+#include "Track.h"
+
+class TextRenderer;
+
+class TimerTrack : public Track {
+ public:
+  explicit TimerTrack(TimeGraph* time_graph);
+  ~TimerTrack() override = default;
+
+  // Pickable
+  void Draw(GlCanvas* canvas, PickingMode picking_mode) override;
+  void OnDrag(int x, int y) override;
+  void OnTimer(const Timer& timer);
+  std::string GetTooltip() const override;
+
+  // Track
+  void UpdatePrimitives(uint64_t min_tick, uint64_t max_tick,
+                        PickingMode picking_mode) override;
```

Comment:
picking_mode is unused.

Reply:
Fixed. I left it in the signature, just marked as unused (commented out), as it is used by EventTrack.